services/softlayer_product_order.go

```diff
@@ -48,3 +48,29 @@ func (slpo *softLayer_Product_Order_Service) PlaceOrder(order datatypes.SoftLaye
 
 	return receipt, nil
 }
+
```

Comment:
So my biggest problem with this now is that when I look at the SL APIs there is no `placeEphemeralDiskOrder` method but rather there is a `placeOrder` method that you can use to place ephemeral disk orders like you are using here.

So I wonder if instead this code should be moved as a utility function in the using code, e.g., `bosh-softlayer-cpi` and changed to just use the `placeOrder` method in `softlaye-go`.

That way `softlaye-go` stays pure in that it only exposes methods that are in the official API docs.

Thoughts?